extractor/contents.py

```diff
@@ -0,0 +1,145 @@
+class Contents:
+    def __init__(self):
+        self.content_list = list()
+
+    def add_content(self, _content):
+        self.content_list.append(_content)
+
+    def __str__(self):
+
+        return_string = '{\n'
+
+        for content in self.content_list:
+            content_string = ' {Index: ' + str(content.index) + '\n'
+            content_string += '  ContentType: ' + content.contentType + '\n'
+            if content.contentType == 'Text':
+                content_string += '  ' + str(content.text)
+            elif content.contentType == 'Image':
+                content_string += '  ' + str(content.image)
+            elif content.contentType == 'Video':
+                content_string += '  ' + str(content.video)
+            elif content.contentType == 'Quote':
+                content_string += '  ' + str(content.quote)
+            elif content.contentType == 'ETweet':
+                content_string += '  ' + str(content.tweet)
+            elif content.contentType == 'EInstagram':
+                content_string += '  ' + str(content.insta)
+            elif content.contentType == 'EPinterestPin':
+                content_string += '  ' + str(content.p_pin)
+            elif content.contentType == 'EYouTube':
+                content_string += '  ' + str(content.youtube)
+
+            return_string += content_string + '\n },\n'
+
+        return return_string + '}'
+
+
+class _Content:
+    counter = 0
+
+    def __init__(self):
+        _Content.counter += 1
+        self.index = _Content.counter
+        self.contentType = ''
+        self.text = Text()
+        self.image = Image()
+        self.video = Video()
+        self.quote = Quote()
+        self.tweet = ETweet()
+        self.insta = EInstagram()
+        self.p_pin = EPinterestPin()
+        self.youtube = EYouTube()
+
+    def set_content_type(self, _type):
+        self.contentType = _type
+
+    def set_text(self, text):
+        self.text = text
+
+
+class Text:
+    def __init__(self):
+        self.type = ''
+        self.text_string = ''
+        self.font_style = ''
+
+    def set_type(self, _type):
+        self.type = _type
+
+    def __str__(self):
+        return '{Text:\n' + '   {type: ' + self.type + '\n   text_string: ' \
+                          + self.text_string + '\n   font: ' + self.font_style + '}\n  }'
+
+
+class Image:
+    def __init__(self):
```

Comment:
Instead of setting empty and then setting. Pass `img_url`, and other properties from constructor and set it initially.

Reply:
Also do it with the rest of the class, i.e. `Video`, `Quote`, `ETweet`, etc whenever it is possible.

Reply:
Made changes accordingly! Updated the constructor for all content types except for Image and Video.